packages/rx_bloc_cli/doc/golden_tests.md

```diff
@@ -0,0 +1,47 @@
+### Smart Widgets dependencies for Golden Tests
+
+In golden tests we want to mock the BLoCs to be able to test all UI states.Using reusable smart widgets that have their own dependency injection will make golden tests to fail. To fix this issue we need to build the smart widgets without injecting the required components during the test. This is why we added `isInTestMode` in `app_constants.dart`.
```

Comment:
A cosmetical suggestion.

We can rephrase the second sentence to something like this:
`By using smart widgets that manage their dependencies, golden tests could fail.`

Reply:
Done